ansible-scylla-node/defaults/main.yml

```diff
@@ -120,7 +120,19 @@ full_inventory: true
 # !!! also make sure you override scylla_yaml_params and remove authorization from there !!!
 scylla_authentication: True
 
+# Set this to false to disable automatic adjustment of system_auth keyspace replication
```

Comment:
Please, add the description of every new variable this patch adds to a patch description.

Also, please, clarify the motivation for this patch in the patch description too (start with it).

Reply:
Done